bokeh/models/tests/test_renderers.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+import unittest
+
+from mock import patch
+
+from bokeh.models.renderers import GlyphRenderer
+from bokeh.plotting import ColumnDataSource, figure
+from bokeh.validation import check_integrity
+
+
+class TestGlyphRenderer(unittest.TestCase):
+    def test_warning_about_colons_in_column_labels(self):
+        sh = ['0', '1:0']
+        p = figure()
+        p.rect('a', 'b', 1, 1, source=ColumnDataSource(data={'a': sh, 'b': sh}))
+        renderer = p.renderers[-1]
```

Comment:
Suggest using `renderer = plot.select({'type': GlyphRenderer})` to get the renderer.